### PR TITLE
improving code base

### PR DESCRIPTION
**Pull request by Faelayis**

### CODE REVIEW

File: libs/scraper.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import DataFileType from './types/DataFile';
+import { configurePuppeteer } from './config';
+import { readJsonFile, delay, writeJsonFile } from './utils';
+import * as fs from "fs/promises"
```
Comment (sctnightcore) [inline libs/scraper.ts:4]: remove fs.

**Review by sctnightcore — CHANGES_REQUESTED** [review]

File: libs/scraper.ts (hunk 1 above)
Comment (Faelayis) [reply libs/scraper.ts:4]: line 35 used

```ts
await fs.writeFile(`./logs/${i["id"]}.html`, pageContent)
```

**Review by sctnightcore — APPROVED** [review]

**Comment by Faelayis** [comment]
should I add eslint, prettier... or not ?🤔

**Comment by sctnightcore** [comment]
> should I add eslint, prettier... or not ?🤔

add it in new pull requests xD

**Comment by Faelayis** [comment]
> > should I add eslint, prettier... or not ?🤔
> 
> add it in new pull requests xD

ok, next pr

**Pull request merged**